graphtools/base.py

```diff
@@ -82,12 +82,21 @@ class Data(Base):
         accepted types: `numpy.ndarray`, `scipy.sparse.spmatrix`.
         `pandas.DataFrame`, `pandas.SparseDataFrame`.
 
-    n_pca : `int` or `None`, optional (default: `None`)
+    n_pca : `int`, `None`, `False`,'True', 'adaptive', optional
```

Comment:
None/False and True/'adaptive' are redundant, so we could leave them out of the first line description. Maybe {`int`, `None`, 'adaptive'}, optional (default: `None`).